### PR TITLE
Fixes #20826: added parsing of server-duids

### PR DESCRIPTION
**Pull request by dmitri-d**

### CODE REVIEW

File: modules/dhcp_common/isc/configuration_parser.rb [hunk 1]
```diff
@@ -382,11 +393,13 @@ def visit_parse_tree_nodes(parse_tree, all_hosts, all_subnets, all_ignored, root
         end
 
         def parse_file(a_path)
+          #File.open(a_path, 'r') {|f| conf.parse!(f.read(:encoding => 'ascii-8bit'), a_path)}
```
Comment (iNecas) [inline modules/dhcp_common/isc/configuration_parser.rb]: are those commented code lines intentional?

**Comment by dmitri-d** [comment]
Removed commented out code.

**Comment by iNecas** [comment]
Works for me: the tests confirm the issue has been covered. @lzap do you want to give it a look, given you reported the original issue?

**Comment by lzap** [comment]
Yeah sorry for late response, looks good.

**Pull request merged**